app/jobs/socure_shadow_mode_proofing_job.rb

```diff
@@ -102,6 +107,7 @@ def build_applicant(
         :state,
         :zipcode,
         :phone,
+        :phone_source, # This shouldn't actually go to Socure
```

Comment:
```suggestion
        :phone_source,
```

(It _doesn't_ go to Socure; that's handled elsewhere.)